src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -419,19 +419,19 @@ export default class ArrayField extends React.Component {
                               <button
                                 type="button"
                                 className="float-left"
-                                aria-label={`${updateText} ${itemName}`}
+                                aria-label={`${updateText} ${itemName} ${itemNumber}`}
                                 onClick={() => this.handleUpdate(index)}
                               >
                                 {updateText}
                               </button>
                             )}
                           </div>
                           <div className="small-6 right columns">
-                            {index !== 0 && (
+                            {multipleRows && (
```

Comment:
Allows you to remove the first indexed item if there is more than one item in the array